portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -53,7 +77,33 @@ public Duration getDuration() {
     return duration;
   }
 
-  public Integer getPriority() {
+  public TaskPriority getPriority() {
+    return priority;
+  }
+
+  public Integer getPriorityInt() {
     return priority.getPriority();
   }
+
+  public long getDurationSeconds() {
+    return duration.getSeconds();
+  }
+
+  public Instant getScheduledTime() {
+    return scheduledTime;
+  }
+
+  // These methods provide a way to tell if two objects are both Task objects
+  // and have all the same fields
+
+  @Override
+  public boolean equals(Object other) {
+    return other instanceof Task && equals(this, (Task) other);
+  }
+
+  private static boolean equals(Task a, Task b) {
+    return a.name.equals(b.name) && a.description.equals(b.description)
```

Comment:
nit: Line break before each `&&`, don't mix lines with one clause and lines with multiple clauses.

Reply:
Done.